Mod2/Lessons/Week2/TDD.md

```diff
@@ -0,0 +1,152 @@
+# Test Driven Development
+
+## Learning Goals
+* Describe the benefits of TDD
+* Practice a TDD workflow
+* Define the difference between Unit and Integration tests
+
+## Warm Up
+
+## Why Write Tests
+
+It can be especially difficult to get started on a new project or even a new iteration of a project. The essence of testing is asking questions and coming up with difficult answers.
```

Comment:
I'm not following this part about asking questions and coming up with difficult answers. Maybe you can add an example here of what you're thinking?

Reply:
Yep - I copied this from the backend lesson - definitely confusing.